sechub-pds-solutions/checkmarx/docker/scripts/checkmarx.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 # SPDX-License-Identifier: MIT
 
-declare -r checkmarx_wrapper="$TOOL_FOLDER/sechub-pds-wrapper-checkmarx.jar"
+declare -r checkmarx_wrapper="$TOOL_FOLDER/sechub-wrapper-checkmarx.jar"
 
 options="";
```

Comment:
Line 6 can be removed. (looks like it comes from a Java programmer because of the `;` :D )
And line 7 can look like this:
```
options="-Dspring.profiles.active=default"
```

Reply:
Was implemebted as you requested.